src/style.css

```diff
@@ -1 +1,81 @@
-/* add styles here */
+@font-face {
+  font-family: 'Avenir Book';
+  src: url('./fonts/Avenir-Book.ttf') format('truetype');
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: 'Avenir Heavy';
+  src: url('./fonts/Avenir-Heavy.ttf') format('truetype');
+  font-weight: bold;
+}
+
+body {
+  font-family: Avenir, sans-serif;
```

Comment:
According to the checklist, remember to use fallback fonts - alternative font-family in case the main one doesn't work. You should provide a list of similar, fallback fonts that can be used if the user's computer doesn't have the first font specified. For example, you can modify this line to: `font-family: Avenir, Arial, sans-serif;`